src/components/layout.js

```diff
@@ -1,14 +1,28 @@
 import Logo from '../assets/logo.png'
 import React from 'react'
 import { Routes } from './routes'
+import { Link } from 'react-router-dom';
 const NavBar = () => (
   <nav
     className="navbar navbar-expand-lg navbar"
-    style={{ backgroundColor: '#243387' }}
+    style={{ backgroundColor: '#243387', color: "#FFFFFF" }}
   >
     <a className="navbar-brand" href="#">
       <img src={Logo} height="50" alt="" />
     </a>
+    <div className="collapse navbar-collapse pull-right" id="navbarSupportedContent">
+      <ul className="navbar-nav mr-auto">
+        {(
+          !window.location.href.includes("main")) ? "" :
```

Comment:
This could be clearer to read if no negation happens. Something along the lines of: 
```suggestion
          window.location.href.includes("main")) ? (<li className="nav-item ">
            <Link to={'/'}>Logout</Link>
          </li>) : ''
```

As an aside, in JS for strings `''` is usually used instead of `""`